e2e/tests/steps_init_test.go

```diff
@@ -158,6 +217,87 @@ func (s *rootSuite) getPrivateKey(
 	return privateKey
 }
 
+// TheApplicationHasAValidEthereumRelaychainAccount fullfils the following condition from feature file:
+//
+//	"Given the application has a valid ethereum relaychain account"
+func (s *rootSuite) TheApplicationHasAValidEthereumRelaychainAccount() {
+	// Account: 0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a   (Arbitrum Bridge)
+	s.relaychains[relaychainEth].account = "0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a"
+}
+
+// TheApplicationHasAValidEthereumRelaychaindHeight fullfils the following condition from feature file:
+//
+//	"Given the application has a valid ethereum relaychain height"
+func (s *rootSuite) TheApplicationHasAValidEthereumRelaychainHeight() {
+	// Ethereum relaychain BlockNumber: 17605670 = 0x10CA426
+	s.relaychains[relaychainEth].height = "0x10CA426"
+}
+
+// TheApplicationHasAValidServicer fullfils the following condition from feature file:
+//
+//	"Given the application has a valid servicer"
+func (s *rootSuite) TheApplicationHasAValidServicer() {
+	s.servicer = servicerA
+}
+
+// An Application requests the account balance of a specific address at a specific height
+func (s *rootSuite) TheApplicationSendsARelayToAServicer() {
+	// ADDPR: Add a servicer staked for the Ethereum RelayChain
```

Comment:
DItto comment above, if this was `ADD_IN_THIS_PR`, it would be very clear of the intention and WIP

Reply:
Thank you for the review. Fixed.